### DISCUSSION TITLE
How do I add two measuring cursors?

### DISCUSSION THREAD

**Discussion by alexandersiemert** [Live-Charts/LiveCharts2#979] [Q&A]
Hello everyone,

I need to create a cartesian chart with two vertical cursors to do some calculations with the chart values at the position from cursor 1 and 2. Did anyone do this already and has an example? Thank you!

**Comment by mpiffari** [comment]
@beto-rodriguez 
Does the library already include an API that exposes cursor functionality?

**Reply by beto-rodriguez** [reply]
You can add a section: https://livecharts.dev/docs/Maui/2.0.0-rc4/samples.general.sections2

**Reply by mpiffari** [reply]
Thank you @beto-rodriguez 
Is it possible to move it with mouse? Is there any way to handle directly via library?
Maybe it seems like the Axes/Crosshairs example: but from source code is not clear which API I need to call/set in order to define the dynamic cursor which follow the mouse pointer.
Otherwise, is there a way to move the cursor, like crosshairs, but, at the end of moving it (like moving it keeping the left-button clicked), it will remain at the same position (like pointed out by @alexandersiemert )

Furthermore, is also possible to detect click interaction over charts?

**Reply by mpiffari** [reply]
@beto-rodriguez thank you for your help and support!

**Reply by parker-white-allient** [reply]
Any update to this?  I'm looking for the exact same functionality.  I tried to follow a crosshairs sample from LiveCharts 1, but many of the functions are deprecated and I felt like I hit a roadblock trying to translate/convert.  I haven't found any suitable examples or resources for how this can be accomplished in LVC2
